account_default_fiscal_position/__openerp__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Default Partner Fiscal Position",
+    "version": "9.0.1.0.0",
+    "author": "Coop IT Easy SCRLfs",
+    "category": "Accounting",
+    "website": "https://www.coopiteasy.be",
+    "depends": ["account"],
```

Comment:
maybe depending on this module could help (or not). If not you should ensure that the vat number is introduce for the partner. Normally to don't pay vat for intracom the vat number is mandatory, valid and should appears on the invoice (which is the case when filled)

Reply:
fixed